src/App.jsx

```diff
@@ -1,22 +1,18 @@
 import "./App.css";
 import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '/vite.svg';
-import './App.css';
 import TodoList from './todolist';
-import AddTodoForm from './AddToDoForm'; // Import the new component
+import AddTodoForm from './addtodoform';
```

Comment:
change import to AddToDoForm from lowercase, throws an import error because file doesn't exist.